content/concepts/discovery/mDNS.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "MDNS"
+description: "MDNS uses a multicast system of DNS records over a local network to enable peer discovery."
+weight: 224
+---
+
+## What is mDNS?
+
+MDNS, or multicast Domain Name System, uses a multicast system of DNS records over a
+local network. Nodes broadcast topics they're interested in into the network instead of
+querying a central name server. The discovery is limited to the peers in the local network.
+
+MDNS uses UDP to enable hostname resolution on local networks, allowing it to be used
+on networks that support multicast, such as an office. In turn, it can also use DNS to
+resolve hostnames to IP addresses on the Internet. MDNS does not require a central DNS
+server to operate, as a node can resolve hostnames to IP addresses in a decentralized
+manner.
```

Comment:
This is not really correct. We're not interested in hostname resolution, we're interested in service discovery. I don't think it has anything to do with DNS over the internet.

I'd go with something along these lines (taken from ChatGPT):
```suggestion
mDNS is short for multicast Domain Name System. It is a way for devices on a local network to discover and communicate with each other without the need for a central server. It is commonly used on home networks to allow devices such as computers, printers, and smart TVs to find each other and connect. It uses a protocol called multicast to broadcast messages on the network, allowing devices to discover each other and exchange information.
```